build/webpack/config_common.js

```diff
@@ -1,28 +1,29 @@
-const UglifyJsPlugin    = require('uglifyjs-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
 const publicPathFactory = require('./helpers').publicPathFactory;
 
 const commonConfig = (grunt) => ({
     devtool: false, // handled by SourceMapDevToolPlugin
     mode   : global.is_release ? 'production' : 'development',
     stats  : {
-        chunks    : false,
-        maxModules: 0,
-        warnings  : false,
+        chunks  : false,
+        warnings: false,
     },
     output: {
         filename     : '[name].js',
         chunkFilename: '[name].min.js',
         publicPath   : publicPathFactory(grunt),
     },
     optimization: {
-        namedChunks: true,
-        minimize   : true,
-        minimizer  : [
-            new UglifyJsPlugin({
-                test     : /\.min\.js/,
-                exclude  : /vendors~/,
-                parallel : true,
-                sourceMap: true,
+        chunkIds : 'named',
```

Comment:
<img width="767" alt="image" src="https://github.com/deriv-com/smarttrader/assets/81898967/cdd8b30b-0ac3-4b62-ab27-9b9f1dd5bfe5">
just as a suggestion we can also use natural for this value as well.
still depends on the requirements that you have in the team and your priorities.

Reply:
@ali-hosseini-deriv I chose 'named' as a replacement to the unsupported property that I had to remove - `namedChunks: true` - in order to keep existing functionality :)